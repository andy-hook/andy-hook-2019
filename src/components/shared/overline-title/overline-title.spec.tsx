import React from "react"
import OverlineTitle from "./overline-title"
import { shallowWithTheme } from "@test-utils"

describe("<OverlineTitle />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "dark",
      <OverlineTitle overline="Overline text">Title text</OverlineTitle>
    )
    expect(tree).toMatchSnapshot()
  })
})
