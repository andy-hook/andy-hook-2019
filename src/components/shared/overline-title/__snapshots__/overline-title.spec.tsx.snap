// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OverlineTitle /> renders correctly 1`] = `ShallowWrapper {}`;
